WalletWasabi.Tests/TestableRpcService.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using WalletWasabi.Gui.Rpc;
+
+namespace WalletWasabi.Tests
+{
+	internal class TestableRpcService
```

Comment:
This class is full of unused parameters.

Reply:
The parameters are there because the tests verify the values in the requests can be matched to those parameters.

Reply:
It's still a "wtf" moment for anyone who encounters this code. Please choose a different design.

Reply:
I cannot find any way to pass parameters by reflection to a method if the that method doesn't have those parameters.

Reply:
There is always a way. Please also don't use reflection.

Reply:
The core of this rpc server uses reflection because it receives a a json request with the method and the parameters to pass to the service method. This is how asp.net does it. Avalonia also uses reflection to know what methods to invoke for commands and bindings.

Please reconsider what you are requesting, I don't think we can do it without redoing the core of this PR and I am pretty sure the alternative design will be really ugly.

Reply:
That is 2 hacks in a high maintenance feature right out of the box and I barely even started to review the code.

Reply:
I am OK with your solution.